src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -4,12 +4,22 @@
  * Add some custom fields that could be only in Truck
  * Do not remove no-field constructor
  */
-public class Truck extends Machine {
+
```

Comment:
Consider adding a JavaDoc comment for the `Truck` class to describe its purpose and any specific details about its fields or behavior. This is important for maintaining good documentation practices.